lib/elixir/lib/io.ex

```diff
@@ -237,12 +237,17 @@ defmodule IO do
   80 characters. The width can be changed by explicitly
   passing the `:width` option.
 
-  See `Inspect.Opts` for a full list of options.
+  The output can be decorated with a tag, by providing `:tag`
+  option to easily distinguish it from other `IO.inspect` calls.
```

Comment:
"by providing **a** tag ..."

Also, we should use `IO.inspect/2` (with the arity 😃).